tests/Amadeus/Client/Struct/Fare/MasterPricerCalendarTest.php

```diff
@@ -106,4 +107,54 @@ public function testCanMakeBasicRequestWithMandatoryElements()
         $this->assertNull($message->travelFlightInfo);
         $this->assertEmpty($message->valueSearch);
     }
+
+    /**
+     * Test creating itinerary/timeDetails/tripDetails.
```

Comment:
Here you should also add a PHPDoc comment to indicate that this doesn't work on Amadeus yet